ci/jenkins/jobs/macros.yaml

```diff
@@ -254,7 +254,29 @@
           return_code=$?
           set -ex
           ./ci/kind/kind-setup.sh destroy "${{FULL_CLUSTER_NAME}}"
-          exit $return_code 
+          exit $return_code
+                   
+- builder:
+    name: builder-kind-flexible-ipam-e2e
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          KIND_TIMEOUT=135
+          IPAM_VLAN_SUBNETS='192.168.241.1/24,192.168.242.1/24'
```

Comment:
For flexible-ipam e2e test, the non-vlan subnet is 192.168.240.1/24, and it is also hardcoded in e2e code. Is it possible to configure this in kind cluster?

Reply:
Yes i configured kind cluster network subnet is in 192.168.240.1/24 in testbed so kind nodes ip will  in 192.168.240.1/24 subnet.